storefront/pages/profile/index.tsx

```diff
@@ -44,21 +49,17 @@ const Profile: NextPage = () => {
       <Head>
         <title>Profile</title>
       </Head>
-      <div className="d-flex justify-content-between pt-5">
+      <div
+        className="d-flex justify-content-between pt-5 col-md-12 mb-2"
+        style={{ height: '100px' }}
+      >
         <h2>User Profile</h2>
-        <button className="btn btn-primary p-0">
-          <Link
-            style={{ width: '100%', display: 'inline-block' }}
-            className={clsx(styles['link-redirect'], 'p-2', 'd-flex', 'align-items-center')}
-            href={{ pathname: '/address' }}
-          >
-            <span style={{ padding: '0 5px 0 0' }}>Go to address list</span>
-            <FaArrowRight />
-          </Link>
-        </button>
       </div>
       <div className="container">
-        <div className="row justify-content-center">
+        <div className="row">
+          <div className="col-md-3 p-0">
+            <UserProfileLeftSideBar type={'profile'} />
```

Comment:
Should a User Profile Layout be created to prevent code duplication so that if we switch pages it's doesnt rerender LeftMenu again, just like out Header and Footer

Reply:
Yes, I will create new issue to create common component to wrap it.